trace/trace.go

```diff
@@ -26,15 +26,21 @@ func init() {
 	rand.Seed(time.Now().Unix())
 }
 
-// (Experimental)
-// If this is set to true,
-// traces will be generated but not actually sent.
-// This should only be set before any traces are generated
-var Disabled bool = false
+// Disabled controls if we'll actually emit the traces.
```

Comment:
Did you mean to remove the `Experimental` tag here? This whole package is technically experimental, but this variable is doubly-so.